examples/authorize_and_store.js

```diff
@@ -20,19 +20,13 @@ async function authorizeAccount(api, pair, who, transactions, bytes) {
     console.log('Transaction authorizeAccount result:', result.toHuman());
 }
 
-async function store(api, pair, data) {
-    console.log('Storing data:', data);
-
+function makeCIDfromData(data) {
```

Comment:
@x3c41a I wanted to do refactors later with Rohit, but ok, let's start now:
- create please `common.js`
- move existing `function cidFromBytes` from `store_chunked_data.js`
- reuse `cidFromBytes`

please, nothing else for now, because we have some other work ongoing, so don't want to cause conflicts